unable to run tests due to invalid @types/node d.ts

**TypeScript Version:**  2.1.5

**Shell command:**

```
$ jake runtests-browser  browser=chrome
```

**Expected behavior:**

tests are executed

**Actual behavior:**
```
tests/webTestServer.ts(318,14): error TS2678: Type '"win64"' is not comparable to type 'Platform'.
```

really, [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/node/index.d.ts#L350) has no definition for "win64", yet it is used in [tests/webTestServer.ts#L318](https://github.com/Microsoft/TypeScript/blob/master/tests/webTestServer.ts#L318)


however, it could be wrong to use "win64" as https://nodejs.org/api/os.html#os_os_platform
states that there is no such value.

--- a/tests/webTestServer.ts
+++ b/tests/webTestServer.ts
@@ -315,7 +315,6 @@ if (browser === "chrome") {
     let defaultChromePath = "";
     switch (os.platform()) {
         case "win32":
-        case "win64":
             defaultChromePath = "C:/Program Files (x86)/Google/Chrome/Application/chrome.exe";
             break;
         case "darwin":

